FreshAir/RZFReleaseNotesCheck.m

```diff
@@ -70,7 +70,8 @@ - (void)checkReleaseNotes:(RZFReleaseNotes *)releaseNotes
         BOOL newVersion = [self.appVersion compare:lastVersion options:NSNumericSearch] == NSOrderedAscending;
         BOOL deviceSupported = lastVersion != nil;
         if (newVersion && deviceSupported) {
-            status = RZFAppUpdateStatusNewVersion;
+            BOOL updateIsForced = [self.appVersion compare:releaseNotes.minimumVersion options:NSNumericSearch] == NSOrderedAscending;
+            status = updateIsForced ? RZFAppUpdateStatusNewVersionForced : RZFAppUpdateStatusNewVersion;
```

Comment:
I believe the forced update check should be pulled out of the guard here. This is guarded by the lastRelease being supported on the device, but the force update check should happen regardless.